### PR TITLE
Remove jekyll build and incomplete sections.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,24 @@
----
```
Comment (schmmd) [inline README.md]: The README was a duplicate of `github.com/allenai/beaker/`

File: docs/concept/groups.md [hunk 2]
```diff
@@ -19,27 +12,27 @@ Each group's detail page shows a table of all experiments, and their tasks, incl
...
-    [
-        {
-            "foo": "bar"
-        },
-        {
-            "baz": 13
-        }
-    ]
-    ```
+```json
```
Comment (ckarenz) [inline docs/concept/groups.md]: This should stay indented so it appears as a block under the bullet.

File: docs/start/image.md [hunk 3]
```diff
@@ -46,16 +40,21 @@ For this tutorial, use the code from [https://github.com/beaker/mnist-example](h
 1. Download/clone https://github.com/beaker/mnist-example. An easy way to do this is to click **Clone or download** from the GitHub page and the **Download** to download the ZIP archive. You'll want to run this code from a directory where you have read/write permission, so if needed, extract the ZIP to such a location (for example, to `~/Documents/mnist-example-master`). 
 
 2. Download the dataset from from [here](https://beaker.org/ds/ds_kf6v919aq7hk/details) to a /data subdirectory of your `mnist-example-master` directory (for example, `~/Documents/mnist-example-master/data`). The *dataset* contains the files, or directories of files, referenced by the code of the experiment. A convenient way to download the MNIST dataset from Beaker is to go to your `mnist-example-master` directory from your Terminal shell, then run: 
+
 ```bash
 $ beaker dataset fetch --output=./data ds_kf6v919aq7hk
```
Comment (ckarenz) [inline docs/start/image.md]: We should use a fully qualified name here and remove the instructions about getting an ID.

File: docs/start/image.md [hunk 4]
```diff
@@ -68,44 +67,54 @@ t10k-labels-idx1-ubyte
...
 
 2. Set and export the EPOCH environment variable, required by this experiment:
-```bash
-$ EPOCH=10
-$ export EPOCH
-```
+
+    ```bash
+    $ EPOCH=10
+      $ export EPOCH
```
Comment (ckarenz) [inline docs/start/image.md]: We should remove the prompts or drop the `bash` tag.  The prompts interfere with formatting.

File: docs/start/image.md [hunk 5]
```diff
@@ -68,44 +67,54 @@ t10k-labels-idx1-ubyte
 Some environment variables must be exported for the Python code to run.
 
 1. Export the python path:
-```bash
-$ export PYTHONPATH=.
-```
+
+    ```bash
+    $ export PYTHONPATH=.
```
Comment (ckarenz) [inline docs/start/image.md]: We should remove the prompts or drop the `bash` tag, preferring the first.  The prompts interfere with formatting.

Same feedback applies throughout.

File: docs/start/image.md (hunk 4 above)
Comment (ckarenz) [inline docs/start/image.md]: Indentation is jagged here.

File: docs/concept/groups.md (hunk 2 above)
Comment (schmmd) [reply docs/concept/groups.md]: Oops--missed that `-` also makes a list.

File: docs/start/image.md (hunk 5 above)
Comment (schmmd) [reply docs/start/image.md]: I don't disagree with your recommendation but I mostly did the latter for ease.  Happy if we revisit over time... Good call.

File: docs/start/image.md (hunk 4 above)
Comment (schmmd) [reply docs/start/image.md]: Weird--either this is a GitHub bug or there's a hidden character lurking.  `:list` didn't expose it though.

**Pull request merged**